src/app/administrator/override/override.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit, OnChanges } from "@angular/core";
+import { OLmsService } from "src/app/common/o/services/oLms.service";
+import { environment } from "src/environments/environment";
+import * as moment from "moment";
+
+@Component({
+  selector: "app-override",
+  templateUrl: "./override.component.html",
+  styleUrls: ["./override.component.css"],
+})
+export class OverrideComponent implements OnInit {
+  overridableLoans: any[];
+
+  constructor(private lmsService: OLmsService) {}
+
+  ngOnInit() {
+    this.readOverridableLoans();
+  }
+
+  readOverridableLoans() {
+    this.lmsService
+      .get(
+        `${environment.adminBackendUrl}${environment.readOverridableLoansUri}`
+      )
+      .subscribe(
+        (result: object[]) => {
+          this.overridableLoans = result;
+          for (const loan of this.overridableLoans)
+            this.lmsService.processLoan(loan);
+        },
+        (error) => {
+          this.overridableLoans = [];
+          // do something with a logger here
+          alert(error);
+        }
+      );
+  }
+
+  doOverride(loan: any) {
+    const dateOut = moment(loan.dateOut).format("YYYY-MM-DDTHH_mm_ss"),
+      overrideUri = `/loans/book/${loan.bookId}/borrower/${loan.cardNo}/branch/${loan.branchId}/dateout/${dateOut}`;
```

Comment:
Did you always process the override loans with a request parameters or did you have to change it from using the request body to parameters to get it to work? just curious.

Reply:
I always used URI parameters.